integrations/extensions/starter-kits/coveo/coveo.openapi.json

```diff
@@ -0,0 +1,403 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "Custom Search Coveo",
+    "description": "Custom Search unofficial partial spec from IBM.  We made this by running the API and observing the behavior.  There is a detailed documentation of the full API at https://docs.coveo.com/en/13/cloud-v2-api-reference/search-api#tag/Search-V2 that you may want to use to add more functionality to this spec.  This spec is a minimal starting point that can handle some basic use cases.",
+    "version": "1.0"
+  },
+  "servers": [
+    {
+      "url": "https://platform.cloud.coveo.com"
+    }
+  ],
+  "paths": {
+    "/rest/search/v2": {
+      "get": {
+        "security": [
+          {
+            "bearerAuth": []
+          }
+        ],
+        "description": "Auto generated using Swagger Inspector",
```

Comment:
I'd prefer not to have "Auto generated using Swagger Inspector" as a description.  Can you write a better description where this occurs?